handler.go

```diff
@@ -789,6 +792,44 @@ func (h *Handler) handleGetFrameViews(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// handleDeleteView handles Delete /frame/view request.
+func (h *Handler) handleDeleteView(w http.ResponseWriter, r *http.Request) {
+	indexName := mux.Vars(r)["index"]
+	frameName := mux.Vars(r)["frame"]
+	viewName := mux.Vars(r)["view"]
+
+	// Retrieve frame.
+	f := h.Holder.Frame(indexName, frameName)
+	if f == nil {
+		http.Error(w, ErrFrameNotFound.Error(), http.StatusNotFound)
+		return
+	}
+
+	// Delete the view.
+	if err := f.DeleteView(viewName); err != nil {
```

Comment:
There is a potential issue here. In a multi-node cluster, when a view does not reside on all nodes then a delete request sent to a node might return `invalid view` when the view is actually valid, there just isn't data for that view on that node.

You can see this by running the following in a 3-node cluster:
```
PIL="http://localhost:10101"
curl -X POST "$PIL/index/i" -d '{"options":{"columnLabel":"col"}}'
curl -X POST "$PIL/index/i/frame/f" -d '{"options":{"rowLabel":"row"}}'
curl -X POST "$PIL/index/i/query" -d 'SetBit(frame=f, row=1, col=100)'
```
results in the following:
```
├── pil0
│   └── i
│       └── f
│           └── .schema
├── pil1
│   └── i
│       └── f
│           └── .schema
└── pil2
    └── i
        └── f
            ├── .schema
            └── views
                └── standard
                    └── fragments
                        └── 0
```
Sending a DELETE view request to either node0 or node1 will not have the expected result.

We could leave it the way it is, and just make it a requirement that DELETE view requests must be sent to all nodes in order to guarantee a delete, or we could change this error so that it just logs but doesn't stop processing (and broadcasts the delete message to other nodes).